packages/cfd/src/Containers/__tests__/compare-accounts-content.spec.js

```diff
@@ -0,0 +1,202 @@
+    const all_landing_companies = {
+        mt_gaming_company: {
+            financial: 'USD',
+        },
+        mt_financial_company: {
+            financial: 'USD',
+            financial_stp: 'USD',
+        },
+    };
+
+    let mock_props;
+
+    beforeEach(() => {
+        mock_props = {
+            is_logged_in: true,
+            platform: 'mt5',
+            show_eu_related: 'false',
+            residence: 'idn',
+            is_eu: 'false',
+            is_uk: 'false',
+        };
+    });
+
+    it('should render the proper column headers properly', () => {
+        render(<CompareAccountsContent {...mock_props} landing_companies={all_landing_companies} />);
+        expect(screen.getByText('Synthetic')).toBeInTheDocument();
+        expect(screen.getByText('CFDs')).toBeInTheDocument();
+        expect(screen.getByText('Financial STP')).toBeInTheDocument();
+    });
+
+    it('should render the proper data', () => {
+        render(<CompareAccountsContent {...mock_props} landing_companies={all_landing_companies} />);
+        expect(screen.getByText(/Account Currency/i)).toBeInTheDocument();
+        expect(screen.getByText('EUR')).toBeInTheDocument();
+        expect(screen.getByText('EUR/GBP/USD')).toBeInTheDocument();
+        expect(screen.getByText('USD')).toBeInTheDocument();
+
+        expect(screen.getAllByText(/Maximum leverage/i)[0]).toBeInTheDocument();
+        expect(screen.getAllByText(/1/i)[0]).toBeInTheDocument();
+        expect(screen.getByText(/Up to 1:1000/i)).toBeInTheDocument();
+        expect(screen.getByText(/Up to 1:30/i)).toBeInTheDocument();
+        expect(screen.getAllByText(/Up to 1:100/i)[0]).toBeInTheDocument();
+
+        expect(screen.getAllByText(/Order execution/i)[0]).toBeInTheDocument();
+        expect(screen.getAllByText(/2/i)[0]).toBeInTheDocument();
+        expect(screen.getAllByText(/Market/i)[0]).toBeInTheDocument();
+        expect(screen.getAllByText(/Market/i)[1]).toBeInTheDocument();
+        expect(screen.getAllByText(/Market/i)[2]).toBeInTheDocument();
+
+        expect(screen.getAllByText(/Spread/i)[0]).toBeInTheDocument();
+        expect(screen.getAllByText('Fixed/Variable')[0]).toBeInTheDocument();
+        expect(screen.getAllByText('Fixed/Variable')[1]).toBeInTheDocument();
+        expect(screen.getAllByText('Variable')[0]).toBeInTheDocument();
+
+        expect(screen.getAllByText(/Commission/i)[0]).toBeInTheDocument();
+        expect(screen.getAllByText(/4/i)[0]).toBeInTheDocument();
+        expect(screen.getAllByText(/No/i)[1]).toBeInTheDocument();
+        expect(screen.getAllByText(/No/i)[2]).toBeInTheDocument();
+        expect(screen.getAllByText(/No/i)[3]).toBeInTheDocument();
+
+        expect(screen.getAllByText(/Minimum deposit/i)[0]).toBeInTheDocument();
+        expect(screen.getAllByText(/No/i)[0]).toBeInTheDocument();
+        expect(screen.getAllByText(/No/i)[1]).toBeInTheDocument();
+        expect(screen.getAllByText(/No/i)[2]).toBeInTheDocument();
```

Comment:
I think this is not a best way to check the existence of text. You have tested text "No" in line 74. How can we make sure this test is not taking that values

Reply:
Will work on it